BankSDK/GiniBankSDK/Sources/GiniBankSDK/Resources/GiniColorScheme.swift

```diff
@@ -0,0 +1,256 @@
+//
+//  GiniColorSchemeV2.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+import GiniCaptureSDK
+
+public struct GiniColorScheme {
+    struct Bg {
```

Comment:
[request] Please don't use abbreviations. Replace `bg` everywhere with `background`.
[clarification] Do we really need this struct if we have for every other struct a bg color?

Reply:
Its a background color for root view and not related to any component like textfield/button. I think it was a reason Anna created it as a separated component